src/pages/waypoint/commands/[[...page]].tsx

```diff
@@ -3,19 +3,27 @@ import waypointData from 'data/waypoint.json'
 import { Product } from 'types/products'
 import { getStaticGenerationFunctions } from 'layouts/docs/server'
 import DocsLayout from 'layouts/docs'
-import DocsPage from 'components/docs-page'
+import DocsView from 'views/docs-view'
+import Placement from 'components/author-primitives/shared/placement-table'
+import NestedNode from 'components/author-primitives/waypoint/nested-node'
 
 const basePath = 'commands'
+const baseName = 'Commands'
```

Comment:
This is somewhat unrelated to this PR's contents, but the lack of configuration option for `baseName` means the breadcrumb bar gets confusing - for example [the `/commands` page shows "Docs" in the breadcrumb bar](https://developer.hashi-mktg.com/waypoint/commands).